### PR TITLE
Future Proofing for the next update

### PR DESCRIPTION
**Pull request by swaggio7**
I have decided to put Recommended Mods 2.0 on hold since I couldn't share the survey. Changes:
-Removed Included Mods and Recommended Mods since we will not need them
-Added VulkanMod to Notes

### CODE REVIEW

**Comment by swaggio7** [comment]
This PR isn't complete though. The basic changes are made, but there maybe some more things to add to the README.

**Comment by swaggio7** [comment]
I'm thinking about adding a features section to the README. Thoughts?

**Comment by swaggio7** [comment]
And with that, Future Proofing reaches its homestrech.

**Comment by swaggio7** [comment]
Shout out to CADIndie and sd! for the image!

**Comment by Flopgop** [comment]
lots of commits time to go through it looking for inaccuracies, spelling mistakes, and grammatical errors :trollface:

**Review by StupidRepo — CHANGES_REQUESTED** [review]
You changed `GitHub` to `Github` [here](https://github.com/QuestCraftPlusPlus/QuestCraft/pull/471/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1) which is wrong. The whole site uses and says `GitHub`, not `Github`. 😃

File: README.md [hunk 1]
```diff
@@ -1,91 +1,98 @@
-# Welcome to the QuestCraft GitHub!
-[Support us on Patreon!](https://patreon.com/QuestCraftXR)
+# Welcome to the QuestCraft Github!
```
Comment (StupidRepo) [inline README.md:1]: `GitHub`, not `Github`!

**Review by StupidRepo — CHANGES_REQUESTED** [review]

**Comment by StupidRepo** [comment]
Thank you!

File: README.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ These are renderers that you can toggle on anytime. Each of them comes with thei
 - ### GL4ES
 GL4ES (**[OpenGL ES](https://www.khronos.org/opengles/#)**) has moderate speed, with moderate mod compatibility. Complex mods like Distant Horizons and Immersive Portals may not work.
 - ### Zink
-**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, while being primarily slow, has great mod compatibility. Complex mods like Distant Horizons and Immersive Portals will work.
+**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, while being primarily slow, has great mod compatibility. Complex mods like Distant Horizons and Immersive Portals shoiuld work.
```
Comment (StupidRepo) [inline README.md:38]: You made a spelling mistake here, BK lover.

**Review by StupidRepo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (swaggio7) [reply README.md:38]: Might need to check your spelling again bud

Comment (StupidRepo) [reply README.md:38]: Who me?

Comment (swaggio7) [reply README.md:38]: image

Comment (StupidRepo) [reply README.md:38]: I'm confused. I didn't put that mistake there, you did.

File: README.md [hunk 3]
```diff
@@ -1,91 +1,98 @@
...
-- [Exordium](https://modrinth.com/mod/exordium) (Decrease GUI framerate to speed up world rendering)
-- [Indium](https://modrinth.com/mod/indium) (Uses the Fabric Rendering API to allow many mods to work with Sodium) (**NOTE: You must have the latest [Sodium](https://modrinth.com/mod/sodium) version in order for the mod to work, otherwise the game WILL crash.**)
-- [LambDynamicLights](https://modrinth.com/mod/lambdynamiclights) (Adds dynamic lights to light emitting blocks. Example, when holding a torch on your hand.)
-- [Mod Menu](https://modrinth.com/mod/modmenu) and [YetAnotherConfigLib](https://modrinth.com/mod/yacl) (Allows you to see and (with YACL) configure installed mods.)
-- [Memory Leak Fix](https://modrinth.com/mod/memoryleakfix) (Fixes random memory leaks to reduce RAM usage)
-- [More Culling](https://modrinth.com/mod/moreculling) (Culls not just leaves, but loads of other things to improve performance. Installing [More Culling Extra](https://modrinth.com/mod/morecullingextra) will also add more culling features!)
+- **[Tweakeroo](https://www.curseforge.com/minecraft/mc-mods/tweakeroo)** (Big QOL mod that changes alot of things)
```
Comment (StupidRepo) [inline README.md:74]: `alot` isn't a word! Change to `a lot`, please :)

**Review by StupidRepo — CHANGES_REQUESTED** [review]

**Comment by StupidRepo** [comment]
As far as I scanned, that should be all spelling mistakes :D
I'll leave you alone now.

File: README.md [hunk 4]
```diff
@@ -1,91 +1,98 @@
...
 
-[Fabric API](https://modrinth.com/mod/fabric-api) (API for Fabric. Essential to basically get any mod running.)
+And boom! You have successfully installed QuestCraft on your VR headset! To play, simply press the **Play** button and wait for it to load, it might take a while depending on headset age and model.
 
-[Ferrite Core](https://modrinth.com/mod/ferrite-core) (Decreases memory usage by doing technical stuff)
+## Supported Renderers (Take note when installing mods!)
+### **We have a [list](https://docs.google.com/spreadsheets/d/12yUTO_HgAEyZFoETxmzqaxPZrOUF1ocaQjbi_u5U2jI/edit?usp=sharing) that contains supported mods that are not in the ModManager! You can also add to this list with this [form](https://forms.gle/TRU7NN4YdgR4oDeW9)!**
+These are renderers that you can toggle on anytime. Each of them comes with their own benefits. **Note that GL4ES and Zink already have performance mods installed. (see [the mod list.)](https://docs.google.com/spreadsheets/d/12yUTO_HgAEyZFoETxmzqaxPZrOUF1ocaQjbi_u5U2jI/edit?usp=sharing)**
```
Comment (StupidRepo) [inline README.md:34]: change `[the mod list.)](...)` to `[the mod list](...).`

**Review by StupidRepo — CHANGES_REQUESTED** [review]

**Comment by swaggio7** [comment]
ia

**Review by StupidRepo — APPROVED** [review]
Once you fix that last little issue, I think this'll be good to merge into the master branch. It's a shame no one has merged it yet though.

**Comment by StupidRepo** [comment]
Well, I spoke too soon. Thanks @CADIndie!

**Comment by swaggio7** [comment]
Dammit, I was going to fix it!

**Pull request merged**